### DISCUSSION TITLE
Can we delete a couple Sourceforge mailing lists?

### DISCUSSION THREAD

**Discussion by dalers** [timschofield/webERP#292] [Development Discussion & Specification]
The webERP.org home page links to the Sourceforge mailing lists, but I suggest the "deleted-70949-cvs" and "web-erp-svn" mailing lists should be deleted for clarity (and of course without knowing why they were created in the first place ;-)  )

![image](https://github.com/user-attachments/assets/0c6cf1b4-5487-40d2-a955-cb7a3240ad32)

**Comment by timschofield** [comment]
You can't actually delete a Sourceforge mailing list. At least you
couldn't last time I looked. The cvs and svn lists were created to
produce an email every time a commit was made to cvs, and then svn.
They are redundant now, but they have mail in them so cannot be
deleted.

Thanks
Tim

On Mon, 9 Dec 2024 at 22:38, Dale Scott ***@***.***> wrote:
>
> The webERP.org home page links to the Sourceforge mailing lists, but I suggest the "deleted-70949-cvs" and "web-erp-svn" mailing lists should be deleted for clarity (and of course not knowing why there were created in the first place ;-) )
>
> image.png (view on web)
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>



-- 
www.weberp.org
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Comment by dalers** [comment]
It may be possible...

From MS Copilot:

To delete a SourceForge mailing list, follow these steps:

1. **Log in to SourceForge**: Ensure you have the necessary administrative privileges for the project.
2. **Access the Mailing List Management**: Navigate to the project's mailing list management page.
3. **Select the Mailing List**: Choose the mailing list you want to delete.
4. **Delete the Mailing List**: Look for an option to delete or remove the mailing list. If this option is not available, you might need to adjust settings in the `mm_cfg.py` file to allow list owners to delete their own lists¹(https://wiki.list.org/pages/viewpage.action?pageId=4030594).

If you prefer using the command line, you can use Mailman's `bin/rmlist` tool:

```bash
bin/rmlist <listname>
```

This command will remove the specified mailing list. Additionally, you may need to manually delete the list's directories and archives if necessary¹(https://wiki.list.org/pages/viewpage.action?pageId=4030594).

If you encounter any issues or need further assistance, feel free to ask!

Source: Conversation with Copilot, [1](https://wiki.list.org/pages/viewpage.action?pageId=4030594)2/10/2024
(1) 4.31. How do I remove a list via the web or command line?. https://wiki.list.org/pages/viewpage.action?pageId=4030594.

**Comment by timschofield** [comment]
The cvs one was actually marked as deleted in the admin window. I have
marked the SVN list as closed (not sure what the difference is.). I don't
think people can subscribe, and if they do they will be disappointed as
there will be no mails.
I think they should be left as is, they are part of the history of the
project.

Tim

On Tue, 10 Dec 2024 at 18:01, Dale Scott ***@***.***> wrote:

> It may be possible...
>
> From MS Copilot:
>
> To delete a SourceForge mailing list, follow these steps:
>
>    1. *Log in to SourceForge*: Ensure you have the necessary
>    administrative privileges for the project.
>    2. *Access the Mailing List Management*: Navigate to the project's
>    mailing list management page.
>    3. *Select the Mailing List*: Choose the mailing list you want to
>    delete.
>    4. *Delete the Mailing List*: Look for an option to delete or remove
>    the mailing list. If this option is not available, you might need to adjust
>    settings in the mm_cfg.py file to allow list owners to delete their
>    own lists¹(https://wiki.list.org/pages/viewpage.action?pageId=4030594).
>
> If you prefer using the command line, you can use Mailman's bin/rmlist
> tool:
>
> bin/rmlist <listname>
>
> This command will remove the specified mailing list. Additionally, you may
> need to manually delete the list's directories and archives if necessary¹(
> https://wiki.list.org/pages/viewpage.action?pageId=4030594).
>
> If you encounter any issues or need further assistance, feel free to ask!
>
> Source: Conversation with Copilot, 1
> <https://wiki.list.org/pages/viewpage.action?pageId=4030594>2/10/2024
> (1) 4.31. How do I remove a list via the web or command line?.
> https://wiki.list.org/pages/viewpage.action?pageId=4030594.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timschofield/webERP/discussions/292#discussioncomment-11524331>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAL6LEMCPSIMA65YRTZAWD2E4T6XAVCNFSM6AAAAABTJ24XPGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNJSGQZTGMI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
www.weberp.org <http://www.weberpafrica.com>
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Comment by dalers** [comment]
Closing discussion after confirming only the web-erp-users and web-erp-developers lists can be subscribed to.

![image](https://github.com/user-attachments/assets/80a3225b-2f47-4d4d-962b-5ff794df440c)
